.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+branches:
+  only:
+    - master
+    - /\bv[0-9]/
+dist: xenial
+python:
+  - "3.4"
+  - "3.5"
+  - "3.6"
+  - "3.7"
+install:
+  - pip install -e .[dev]
+script:
+  - flake8 qatrfm
+  - flake8 tests
+deploy:
```

Comment:
Are we already on pypi?

Reply:
no no, actually I will remove this section for now. We'll see in the future :)